doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -477,8 +477,8 @@ documentation on :ref:`typeclasses` and the theories files in Classes
 for further explanations.
 
 One can inform the rewrite tactic about morphisms and relations just
-by using the typeclass mechanism to declare them using Instance and
-Context vernacular commands. Any object of type Proper (the type of
+by using the typeclass mechanism to declare them using the :cmd:`Instance` and
+:cmd:`Context` vernacular commands. Any object of type Proper (the type of
```

Comment:
Now works for me.

I replaced "vernacular command" with "command" in several places (one in changes.rst).  There are 22 uses of the plural form, mostly in changes.rst--you don't want to change those?  It seems needlessly redundant.

(I feel such a rush of power as I rewrite history.rst. :-) )

"vernacular file" (meaning a .v) -> "proof file" or "proof script file"?

"vernacular notations" (in SSR) - think this is a reference to the Notation commands

"Preformatted vernacular" should be ??

Reply:
> There are 22 uses of the plural form, mostly in changes.rst--you don't want to change those? It seems needlessly redundant.

I'm just reluctant to "rewrite history", but do as you wish.

>"vernacular file" (meaning a .v) -> "proof file" or "proof script file"?

I would say "Coq file".

>"vernacular notations" (in SSR) - think this is a reference to the Notation commands

Yes.

> "Preformatted vernacular" should be ??

"Preformatted Coq code"?

BTW, the paragraph above reads:

>Coq material is quoted between the delimiters [ and ]. Square brackets may be nested, the inner ones being understood as being part of the quoted code (thus you can quote a term like fun x => u by writing [fun x => u]). Inside quotations, the code is pretty-printed in the same way as it is in code parts.

The example of quoting `fun x => u` may sound very strange since it doesn't contain square brackets. Unless you know about the history of Coq syntax. In version 8.0 (pre-syntax-change) of the reference manual (https://coq.inria.fr/distrib/V8.0/doc/), it read:

> Coq material is quoted between the delimiters [ and ]. Square brackets may be nested, the inner ones being understood as being part of the quoted code (thus you can quote a term like [x:T]u by writing [[x:T]u]). Inside quotations, the code is pretty-printed in the same way as it is in code parts.

And when the reference manual was converted to the new syntax, people didn't realize that this example did not make sense anymore!

Reply:
I kept uses of "vernacular" in the changes and history files, mostly only changing titles.  I changed the example back to what it was in V8.0--that's what you wanted, right?

Reply:
Yes, I wanted indeed to go back to an example with square brackets. Your example is not *per se* a Coq term, so what about making it `let id := fun [T : Type] (x : T) => x in id 0`, which is a complete Coq term with a real meaning?

Reply:
comme ça?

(thus you can quote a term like ``let id := fun [T : Type] (x : t) => x in id 0``
by writing  ``[let id := fun [T : Type] (x : t) => x in id 0]``)

Reply:
Yep!